src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeFetchers.scala

```diff
@@ -155,24 +155,27 @@ trait BlocksDataFetchers {
     import JsonDecoders.Circe.Rights._
 
     /** the input type, e.g. ids of data */
-    override type In = BlockHash
+    override type In = (Option[Int], Option[Int], BlockHash)
```

Comment:
Instead of a tuple, could we please use a case class? It will make the code more idiot proof.